docs/content/gameobject/reading-and-writing-component-data.md

```diff
@@ -99,12 +102,16 @@ public class WriteHealthBehaviour : MonoBehaviour
 
 1. Declare a field of type `Health` Writer and decorate it with a `[Require]` attribute. 
 
-2. Register a callback for `Reader.ComponentUpdated(ISpatialComponentUpdate update) +=` or for `Reader.<component property name>Updated() +=` during `OnEnable()`.
+2. Register a callback for `Reader.ComponentUpdated +=` or for `Reader.<component property name>Updated +=` during `OnEnable()`.
     *  `Reader.ComponentUpdated` is invoked when any component property is updated.
     *  `Reader.<component property name>Updated` is invoked when a specific component property is updated.
 
 **Note:** 
-`Reader.ComponentUpdated` callbacks are invoked before specific property update callbacks. Callbacks can be deregistered using `Reader.ComponentUpdated(ISpatialComponentUpdate update) -=` and `Reader.<component property name>Updated() -=`. Callbacks are also automatically deregistered when a Reader or Writer is removed. Do not deregister callbacks during `OnDisable()` as that’s an invalid operation.
+`Reader.ComponentUpdated` callbacks are invoked before specific property update callbacks. Callbacks can be deregistered using `Reader.ComponentUpdated -=` and `Reader.<component property name>Updated -=`. Callbacks are also automatically deregistered when a Reader or Writer is removed. Do not deregister callbacks during `OnDisable()` as that’s an invalid operation.
+
+**Known issue warning:** 
+- At the moment, the `ISpatialComponentUpdate update` argument of `Reader.ComponentUpdated` may indicate that a component property was changed even if it was not truly changed. This is a bug and not intended behaviour and will be fixed in an upcoming update.
```

Comment:
* Don't think we need "at the moment" - we know it's now.
* Think we need present perfect (recent past and we use it more as a description) "has changed" rather than "was changed".
* Go for active voice if we can, so it's clear who is doing what.
* US spelling.

So, suggest...
"The `ISpatialComponentUpdate update` argument of `Reader.ComponentUpdated` may indicate that a component property has changed when it has not. This is unintended behavior which we will fix in an upcoming update."

Reply:
Fixed.